### PR TITLE
Add support for node 18 by updating node-sass

### PR DESCRIPTION
**Pull request by 852Kerfunkle**
And drop support for node 12.

It's probably time, it's the next LTS :)

I have not fully tested it (too reliant on CI maybe), but it works in dev. It's possible it'll also work with node 19, but I did not try.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13493/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13493/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13493/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13493/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13493/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13493/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13493/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13493/merge#BCU1XR#0

**Comment by sebavan** [comment]
> And drop support for node 12.
> 
> It's probably time, it's the next LTS :)
> 
> I have not fully tested it (too reliant on CI maybe), but it works in dev. It's possible it'll also work with node 19, but I did not try.

Was this checked on the forum before ? We can not ship this without an extensive set of tests and compatibility assessment from some of our existing dependencies.

I will let @RaananW check tomorrow.

**Comment by 852Kerfunkle** [comment]
> > And drop support for node 12.
> > It's probably time, it's the next LTS :)
> > I have not fully tested it (too reliant on CI maybe), but it works in dev. It's possible it'll also work with node 19, but I did not try.
> 
> Was this checked on the forum before ? We can not ship this without an extensive set of tests and compatibility assessment from some of our existing dependencies.
> 
> I will let @RaananW check tomorrow.

Fair enough. I needed it, so I PR'd it. It'll have to happen eventually, feel free to treat it as a draft or to reject it.

Cheers.

**Comment by sebavan** [comment]
> > > And drop support for node 12.
> > > It's probably time, it's the next LTS :)
> > > I have not fully tested it (too reliant on CI maybe), but it works in dev. It's possible it'll also work with node 19, but I did not try.
> > 
> > 
> > Was this checked on the forum before ? We can not ship this without an extensive set of tests and compatibility assessment from some of our existing dependencies.
> > I will let @RaananW check tomorrow.
> 
> Fair enough. I needed it, so I PR'd it. It'll have to happen eventually, feel free to treat it as a draft or to reject it.
> 
> Cheers.

It will for sure happen soon knowing @RaananW, I just want to be sure he can validate all the dependencies before merging it.

**Comment by 852Kerfunkle** [comment]
@sebavan Oh by the way on the note of

> Was this checked on the forum before ?

There isn't really a "Development" section on the forum. Either that or I'm not invited :D . It would maybe be useful.

**Comment by sebavan** [comment]
> There isn't really a "Development" section on the forum. Either that or I'm not invited :D . It would maybe be useful.

You are correct, I think those conversations usually end up in either "feature requests" or "questions".

cc @carolhmj our forum GURU

**Comment by 852Kerfunkle** [comment]
> > There isn't really a "Development" section on the forum. Either that or I'm not invited :D . It would maybe be useful.
> 
> You are correct, I think those conversations usually end up in either "feature requests" or "questions".
> 
> cc @carolhmj our forum GURU

Indeed, I also just realised "feature requests" doubles for this stuff. No worries, I'm happy knowing this now!

**Comment by carolhmj** [comment]
Yup Feature Requests works fine for that :)

**Comment by RaananW** [comment]
Thanks so much for the PR.

I would rather test before making changes. I agree - we should fully support node 18. But I am not 100% sure this is the only change.
I'm moving to to draft for now, until I get a change to fully test that

**Review by RaananW — APPROVED** [review]

**Pull request merged**
